CV/Official_Announcement.md

```diff
@@ -0,0 +1,89 @@
+
+Để giúp một số thành viên của hội nắm bắt được kiến thức căn bản về AI/ML, đặc biệt là các bạn đang làm Software Engineer muốn học và tìm hiểu về lĩnh vực này, từ năm 2019, hội VJAI sẽ tổ chức thêm 1 hoạt động mới là các khoá học ngắn về từng chủ đề trong lĩnh vực AI. Chúng ta sẽ bắt đầu với khoá học về chủ đề `Computer Vision and Deep Learning for beginner`.
```

Comment:
Phần này có thể đẩy len Readme của toàn bộ Minicourse và list ra các minicourse mà mình có, đầu tiên là CV ạ.

Reply:
Tạm thời đã đưa đoạn này vào README của repo, rồi link đến file announcement này. Tuy nhiên, đoạn này cần giữ ở đây để đưa lên FB post.

Sau này sẽ làm list các courses cho file README của repo.